### PR TITLE
Remove signature algorithm property

### PR DESCRIPTION
**Pull request by rkondratowicz**
I'm not sure why this property existed in the first place (all environments used `sha256withrsa`)
Keeping the toggle (now just between extracting the inner envelope and using it 'as is') as we may flip it in the future (so that it's consistent with Crime) and then remove it completely.

flux PR: https://github.com/hmcts/cnp-flux-config/pull/3081

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -205,3 +204,5 @@ flyway:
 
 process-payments:
   enabled: ${PROCESS_PAYMENTS_ENABLED:true}
+
+use-wrapping-zip: ${USE_WRAPPING_ZIP:true}
```
Comment (nitinprabhu) [inline src/main/resources/application.yaml:208]: this env var should be ideally defined in values yaml and default value should be removed

File: charts/bulk-scan-processor/values.yaml [hunk 2]
```diff
@@ -19,7 +19,6 @@ java:
     REUPLOAD_DELAY: "4000"
     NOTIFICATIONS_TO_ORCHESTRATOR_TASK_ENABLED: "true"
     NOTIFICATIONS_TO_ORCHESTRATOR_TASK_DELAY: "3000"
-    STORAGE_BLOB_SIGNATURE_ALGORITHM: "sha256withrsa"
```
[the remark this answers is not in the document]
Comment (doncem) [reply charts/bulk-scan-processor/values.yaml:22]: non breaking change. unless pipeline blocks it as i heard there is or suppose to be a rule whenever `values.yaml` changed - must exist version bump

File: src/main/resources/application.yaml (hunk 1 above)
Comment (rkondratowicz) [reply src/main/resources/application.yaml:208]: perhaps I'll just remove the env var then, we currently don't need to change it

File: charts/bulk-scan-processor/values.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (nitinprabhu) [reply charts/bulk-scan-processor/values.yaml:22]: Pipeline blocks it :)

[the remark this answers is not in the document]
Comment (doncem) [reply charts/bulk-scan-processor/values.yaml:22]: good platform people :)

[the remark this answers is not in the document]
Comment (rkondratowicz) [reply charts/bulk-scan-processor/values.yaml:22]: https://github.com/hmcts/cnp-flux-config/pull/3080

**Pull request merged**